paho-client/requirements.txt

```diff
@@ -0,0 +1 @@
+paho-mqtt
```

Comment:
Normally you pin a version here. `pip freeze | grep paho-mqtt > requirements.txt` would do.

Reply:
done